Cake-css.css

```diff
@@ -0,0 +1,123 @@
+.hamIcon{
+    position: absolute;
+    right: 10px;
+    top: 20px;
+}
+
+.logo_image{
+    display: flex;
+    justify-content: center;
+    height: 200px;
+    background: url("images/Mynavbarimage.jpeg");
+    background-position: center;
+}
+.nav_bar{
+    background: url("images/nav_back.JPG");
+   
```

Comment:
Try adding rules for `background-repeat` and `background-size` here to make the image cover the whole nav